CHANGELOG.md

```diff
@@ -84,6 +84,11 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - `ProtobufConvert` has been implemented for `MapProof`. (#1512)
 
+- New variant of the `ProofMapIndex` have been introduced - `RawProofMapIndex`.
+  It is used for keys that maps directly to `ProofPath`, for example `Hash` and
+  `PublicKey`. By default `ProofMapIndex` is used for keys that implements
+  `ObjectsHash`. (#1531)
```

Comment:
Do I understand correctly that both key types result in the same result for `Hash` keys? It may make sense to mention it and possibly introduce an alias for `ProofMapIndex<T, Hash, V>` (like `HashMapIndex<T, V>`) to make this more transparent. (Although it's up to discussion whether this actually simplifies things.)